src/include/OSL/device_string.h

```diff
@@ -92,27 +92,27 @@ struct DeviceString {
 // DeviceString instead of ustring for some input parameters, so we use this
 // typedef to select the correct type depending on the target.
 #ifndef __CUDA_ARCH__
-typedef OIIO::ustring StringParam;
+typedef ustring StringParam;
 #else
 typedef DeviceString StringParam;
 #endif
 
 
-OSL_NAMESPACE_EXIT
-
-
 #ifdef __CUDA_ARCH__
-namespace OSLDeviceStrings {
+namespace DeviceStrings {
 #define STRDECL(str,var_name)                       \
-    extern __device__ OSL::DeviceString var_name;
+    extern __device__ OSL_NAMESPACE::DeviceString var_name;
 #include <OSL/strdecls.h>
 #undef STRDECL
 }
 #endif
 
 
+OSL_NAMESPACE_EXIT
+
+
 #ifdef __CUDA_ARCH__
-namespace StringParams = OSLDeviceStrings;
+namespace StringParams = OSL_NAMESPACE::DeviceStrings;
 #else
-namespace StringParams = OSL::Strings;
+namespace StringParams = OSL_NAMESPACE::Strings;
```

Comment:
Why this change?

Reply:
Was fighting custom-namespacing for a moment, that may be collateral damage and unnecessary.

Reply:
Makes sense, thanks.